src/PureScript/Backend/Chez/Convert.purs

```diff
@@ -179,11 +181,23 @@ codegenExpr codegenEnv@{ currentModule } s = case unwrap s of
     S.runUncurriedFn
       (S.Identifier $ rtPrefixed "array-ref")
       [ codegenExpr codegenEnv e, S.Integer $ wrap $ show i ]
+  Accessor e
+    (GetCtorField (Qualified (Just (ModuleName "Data.List.Types")) (Ident "Cons")) _ _ _ "value0" _) ->
```

Comment:
I personally would match on the field index (the last slot), rather than the field name. The compiler generates a field name, but it's kind of nonsense IMO.

Reply:
Perfect. I was wondering whether I can trust this name or not.